lib/LTP/utils.pm

```diff
@@ -152,7 +152,8 @@ sub export_ltp_env {
 
 # Set up basic shell environment for running LTP tests
 sub prepare_ltp_env {
-    assert_script_run('export LTPROOT=' . get_ltproot() . '; export LTP_COLORIZE_OUTPUT=n TMPDIR=/tmp PATH=$LTPROOT/testcases/bin:$PATH');
+    assert_script_run('export LTPROOT=' . get_ltproot() . '; export LTP_COLORIZE_OUTPUT=n ' .
+          ' TMPDIR=/tmp PATH=$LTPROOT/testcases/bin:$LTPROOT/testcases/realtime/func/sched_football/:$PATH');
```

Comment:
Maybe we should have some conditional `record_info` or comment here. Otherwise LGTM.

Reply:
I added a comment why `$LTPROOT/testcases/realtime/func/sched_football/` was added. If you consider `record_info()`, I can add it (I would prefer unconditionally).

Reply:
I'm fine. @mdoucha could you please check as well?